cfme/configure/tasks.py

```diff
@@ -252,3 +257,9 @@ def is_analysis_finished(name, task_type='vm', clear_tasks_after_success=True):
                             task_name=tabs_data['task'].format(name),
                             expected_status=tabs_data['state'],
                             clear_tasks_after_success=clear_tasks_after_success)
+
+
+def wait_for_analysis_finish(task_name, delay, timeout):
```

Comment:
Also, this should be called `wait_analysis_finished`, you can check out naming of functions above (`is_analysis_finished`, `is_task_finished`, ...)